calls.go

```diff
@@ -82,14 +106,42 @@ func (call *callsCmd) list(ctx *cli.Context) error {
 		App:     app,
 		Context: context.Background(),
 	}
-	if ctx.Args().Get(1) != "" {
-		route := ctx.Args().Get(1)
-		params.Route = &route
+	if ctx.String("cursor") != "" {
+		cursor := ctx.String("cursor")
+		params.Cursor = &cursor
 	}
+	if ctx.String("path") != "" {
+		route := ctx.String("path")
+		params.Path = &route
+	}
+	if ctx.String("from-time") != "" {
+		fromTime := ctx.String("from-time")
+		fromTime_int64, err := time.Parse(time.RFC3339, fromTime)
+		if err != nil {
+			return err
+		}
+		res := fromTime_int64.UnixNano() / int64(time.Second)
```

Comment:
can just do `Unix()` here, no need to take nanos and divide seconds out.

Reply:
addressed